api/src/main/java/org/openmrs/util/OpenmrsClassLoader.java

```diff
@@ -590,26 +585,19 @@ protected static void nullInstance(Object instance) {
 					if (null != value) {
 						Class<?> valueClass = value.getClass();
 						if (!loadedByThisOrChild(valueClass)) {
-							if (log.isDebugEnabled()) {
-								log.debug("Not setting field " + field.getName() + " to null in object of class "
-										+ instance.getClass().getName() + " because the referenced object was of type "
-										+ valueClass.getName() + " which was not loaded by this WebappClassLoader.");
-							}
+							log.debug(
+							    "Not setting field {} to null in object of class {} because the referenced object was of type {} which was not loaded by this WebappClassLoader. {}",
```

Comment:
Are you passing in 3 or 4 arguments?

Reply:
I have made the modifications on the ticket.